lib/modules/hosts/imgur.js

```diff
@@ -25,6 +25,33 @@ const imgur = {
 				value: 'direct',
 			}],
 		},
+		imgurImageResolution: {
+			description: 'Which size of imgur images should be loaded? (Great for metered internet)',
+			type: 'enum',
+			value: '', // image id suffix
```

Comment:
Next iteration, I'd like to default to large and only load full when resizing up.